### PR TITLE
Add ogl command

### PR DESCRIPTION
**Pull request by ssunil3232**
fixes #47

### CODE REVIEW

File: src/main/java/seedu/address/storage/OglList.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+     */
+    public static boolean hasOgl (String toFind) {
+        return oglNames.contains(toFind);
+    }
+    /**
+     * Prints the list
+     */
+    public static void listOgl() {
+        for (int i = 0; i < ogls.size(); i++) {
+            System.out.println(ogls.get(i));
```
Comment (shanseet) [inline src/main/java/seedu/address/storage/OglList.java:47]: Not sure if  list can be printed out like this. Need to pass PersonCard and show from UI?

Comment (ssunil3232) [reply src/main/java/seedu/address/storage/OglList.java:47]: Yup this is just dummy code. Will update in v1.3

File: src/main/java/seedu/address/logic/commands/AddOglCommand.java [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+            + PREFIX_BIRTHDAY + "27071999 "
+            + PREFIX_PHONE + "98765432 "
+            + PREFIX_EMAIL + "johnd@example.com "
+            + PREFIX_MAJOR + "Information System "
+            + PREFIX_GROUP + "G1 "
+            + PREFIX_TAG + "friends "
+            + PREFIX_TAG + "owesMoney";
+
+    public static final String MESSAGE_SUCCESS = "New ogl added: %1$s";
+    private static final String MESSAGE_DUPLICATE_FRESHMAN = "This ogl already exists in the address book";
```
Comment (tantantan277) [inline src/main/java/seedu/address/logic/commands/AddOglCommand.java]: Shouldn't it be MESSAGE_DUPLICATE_OGL ?

File: src/main/java/seedu/address/storage/OglList.java [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    /**
+     * Checks if a freshman exists
+     */
+    public static boolean hasOgl (String toFind) {
+        return oglNames.contains(toFind);
+    }
+    /**
+     * Prints the list
+     */
+    public static void listOgl() {
```
Comment (tantantan277) [inline src/main/java/seedu/address/storage/OglList.java:45]: Is this function still neccessary?

File: src/main/java/seedu/address/logic/commands/AddOglCommand.java (hunk 2 above)
Comment (ssunil3232) [reply src/main/java/seedu/address/logic/commands/AddOglCommand.java]: Thanks for pointing this out!:)

File: src/main/java/seedu/address/storage/OglList.java (hunk 3 above)
Comment (ssunil3232) [reply src/main/java/seedu/address/storage/OglList.java:45]: will resolve in the next PR

Comment (tantantan277) [reply src/main/java/seedu/address/storage/OglList.java:45]: Just curious how are you gonna show the list ?

**Comment by shanseet** [comment]
fixes #45

**Pull request merged**